integration/plugin/common/main_test.go

```diff
@@ -0,0 +1,33 @@
+package common // import "github.com/docker/docker/integration/plugin/common"
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/docker/docker/internal/test/environment"
+)
+
+var testEnv *environment.Execution
+
+func TestMain(m *testing.M) {
+	var err error
+	testEnv, err = environment.New()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	//err = environment.EnsureFrozenImagesLinux(testEnv)
+	//if err != nil {
```

Comment:
?

Reply:
oh! hm 🤔 LOL I think I did that because we didn't need images in these tests, so I wanted to speed things up a bit, but I should probably either remove it, or put it back (in case other tests are added)

Reply:
Un-commented it; guess it's better to keep it consistent with the other suites for now

Reply:
If it's not needed, why keep it?